deploy/template.yaml

```diff
@@ -20,9 +20,6 @@ Parameters:
   ImageTag:
     Description: The tag of passport-front image to deploy in the task definition.
     Type: String
-  ApiBaseUrl:
```

Comment:
So you might want to check this out. I think if you remove it but Concourse is still setting this parameter then the update command fails because its being provided a parameter which doesn't exist in the template. When I did it I left this in (accept the CFN-lint warning for a bit) and then removed it in coordination with a change to the Concourse pipeline.